Pod/Classes/GoogleMaterialIconFont.swift

```diff
@@ -49,8 +49,8 @@ private class FontLoader {
         }
         
         let data = NSData(contentsOfURL: fontURL)
-        let provider = CGDataProviderCreateWithCFData(data)
-        let font = CGFontCreateWithDataProvider(provider)!
+        let provider = CGDataProviderCreateWithCFData(data!)
+        let font = CGFontCreateWithDataProvider(provider!)
```

Comment:
Can you follow this way? I want to make it safer just in case.
https://github.com/kitasuke/GoogleMaterialIconFont/pull/20/files#diff-3911d9af4605e9b572c4abe9a1209f49R55